.devcontainer/dev.code-workspace

```diff
@@ -0,0 +1,57 @@
+// !/bin/bash
+//  SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+{
+    "folders": [
+        {
+            "path": ".."
+        }
+    ],
+    "launch": {
+        "version": "0.2.0",
+        "configurations": [
+			{
+				"name": "abp_pcap_detection",
+				"type": "python",
+				"request": "launch",
+				"program": "${workspaceFolder}/examples/abp_pcap_detection/run.py",
+				"console": "integratedTerminal",
+				"cwd": "${workspaceFolder}/examples/abp_pcap_detection",
+				"justMyCode": false,
+				"args": [
+					"--input_file=${workspaceFolder}/examples/data/pcap_dump.jsonlines",
+					"--output_file=${workspaceFolder}/.tmp/pcap_output.jsonlines",
+					"--server_url=triton:8001",
+					"--model_name=abp-pcap-xgb"
+				]
+			},
+            {
+                "name": "(gdb) Attach",
+                "type": "cppdbg",
+                "request": "attach",
+                "program": "${workspaceRoot}/.cache/conda/envs/morpheus/bin/python",
+                "processId": "${command:pickProcess}",
+                "MIMode": "gdb",
+                "setupCommands": [
+                    {
+                        "description": "Enable pretty-printing for gdb",
+                        "text": "-enable-pretty-printing",
+                        "ignoreFailures": true
+                    }
+                ]
+            }
+        ]
+    }
+}
```

Comment:
Can we unify the `dec.code-workspace` with the root `morpheus.code-workspace`? Having two workspace files will only be problematic.

Reply:
I've deleted the devcontainer workspace, and updated the abp pcap example to save the output to a `.tmp` dir.